thrift-logger/src/main/java/com/pinterest/singer/client/BaseThriftLoggerFactory.java

```diff
@@ -34,6 +35,16 @@
   protected ConcurrentHashMap<String, ThriftLogger> loggersByTopic
       = new ConcurrentHashMap<String, ThriftLogger>();
 
+  private int sleepInSecBeforeCloseLoggers = -1;
```

Comment:
It seems this number gets converted to ms value later. Why not define this as `sleepInMsBeforeCloseLoggers`?

Reply:
In singer-common module ( https://github.com/pinterest/singer/blob/master/singer-commons/src/main/java/com/pinterest/singer/loggingaudit/client/AuditEventKafkaSender.java#L82 ),  we have similar parameter (in seconds) to control the shutdown behavior.   I feel it is better to be consistent in naming across modules.